R/bundle.R

```diff
@@ -80,7 +82,24 @@ bundleApp <- function(appName, appDir, appFiles, appPrimaryDoc, assetTypeName,
   bundlePath
 }
 
-isShinyRmd <- function(filename) {
+appHasParameters <- function(appDir, files) {
+  rmdFiles <- grep("^[^/\\\\]+\\.rmd$", files, ignore.case = TRUE, perl = TRUE,
```

Comment:
Ah. We only support single-doc rendering. It's fine to have multiple HTML artifacts, but we only render a single Rmd at present. I suppose when we eventually have multiple docs, we might want metadata about each file.